### ISSUE TITLE
[BUG] : Steam login page doesnt open on linux

### ISSUE THREAD

**Issue by MasterDomino** [Zagrios/bs-manager#571]
## Bug Description

upon trying to download a version of beat saber a window asking to login to steam with a qr code should popup and allow to login, but it immediately closes

## Reproduction Steps
1. start from AppImage
2. try to download any beat saber version
3. select steam

## Expected Behavior
a popup window should stay on the screen and allow for user to login on their platform

## System Specs
-   OS: Debian 12 Bookworm, Kernel 6.1.0-23-amd64, KDE Plasma 5.27.5
-   Version: BSManager 1.5.0 alpha 3

## Additional Context
upon a further analysis, it seems like now BSManager requires a writable file-system so it might completely be my fault here that i have missconfigured something beforehand

## Logs
im adding a snippet from logs that i think might be causing this behavior:
```
[2024-08-30 16:06:34.970] [error] Unhandled Exception UnhandledRejection Error: EROFS: read-only file system, open '/tmp/.mount_BSManaDl0g0c/resources/assets/jsons/bs-versions.json'
    at writeFileSync (node:fs:2368:20)
    at updateLocalVersions (/tmp/.mount_BSManaDl0g0c/resources/app.asar/dist/main/main.js:14:1150922)
    at u.loadBsVersions (/tmp/.mount_BSManaDl0g0c/resources/app.asar/dist/main/main.js:14:1151187)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at u.getAvailableVersions (/tmp/.mount_BSManaDl0g0c/resources/app.asar/dist/main/main.js:14:1151282)
[2024-08-30 16:06:34.971] [info]  Oculus library auto-detection not supported on non-windows platforms
[2024-08-30 16:06:34.971] [info]  Oculus library auto-detection not supported on non-windows platforms
[2024-08-30 16:06:34.972] [info]  Finded versions folders []
[2024-08-30 16:06:54.202] [info]  DepotDownloader process end with code null
[2024-08-30 16:06:54.203] [error] initVersionMetadata error Error: ENOENT: no such file or directory, open '/home/void/BSManager/BSInstances/1.37.1/metadata.config'
[2024-08-30 16:06:54.203] [info]  Oculus library auto-detection not supported on non-windows platforms
[2024-08-30 16:06:54.203] [info]  Oculus library auto-detection not supported on non-windows platforms
[2024-08-30 16:06:54.204] [info]  Finded versions folders []
[2024-08-30 16:06:56.083] [info]  DepotDownloader process end with code null
```

**Comment by MasterDomino** [comment]
addition to my logs, this was already mentioned in #226 so might be completely irrelevant to the actual issue at hand

**Comment by silentrald** [comment]
Hi @MasterDomino, the issue described, the one with the login modal not showing, is already patched although it's not yet part of 1.5.0-alpha-3 release. You might need to download the version within the latest version from github actions. You can access it by going to github actions and searching for the build or just access this link https://github.com/Zagrios/bs-manager/actions/runs/10619924192 and download the "release-ubuntu-latest".

**Comment by MasterDomino** [comment]
@silentrald, can confirm it now works, closing the issue.
